testsuite/docs-examples-cpp/ref/out.txt

```diff
@@ -1,3 +1,15 @@
+error: Uninitialized input image
+error: Uninitialized input image
```

Comment:
I'm a little concerned about these lines... is something in an example not working?

Reply:
Oh, never mind, those are from the ones where you are purposely making an error and showing how to check it, right?

Reply:
This is the result of testing the error outputs:
https://openimageio.readthedocs.io/en/v2.5.4.0/imagebufalgo.html#return-values-and-error-messages

I reproduced the tests, which print out "error: ..." for the two first examples:
https://github.com/buddly27/OpenImageIO/blob/pr3992/testsuite/docs-examples-cpp/src/docs-examples-imagebufalgo.cpp#L46
https://github.com/buddly27/OpenImageIO/blob/pr3992/testsuite/docs-examples-cpp/src/docs-examples-imagebufalgo.cpp#L59

This could potentially be tested differently:
```cpp
// BEGIN-imagebufalgo-output-error2
// Method 2: Write into an existing image
ImageBuf dst;   // will be the output image
bool ok = ImageBufAlgo::over (dst, fg, bg);
std::string error;
if (! ok)
    error = dst.geterror();
// END-imagebufalgo-output-error2

static_assert(!error.empty(), "An error should have been set");
```

Would you prefer that?

Reply:
No, I think it's fine as it is. I just saw the error message in the captured output and momentarily forgot that this was expected.